src/Artsy/SystemContext.tsx

```diff
@@ -17,12 +16,6 @@ export interface SystemProps {
    */
   user?: User
 
-  /**
-   * Media queries that are passed to Responsive when an app first mounts. Useful
-   * for SSR rendering views for mobile devices.
-   */
-  initialMatchingMediaQueries?: MatchingMediaQueries
```

Comment:
This probably relates to my question above -- whats the thinking in removing from here?

Reply:
I wanted to capture detection of devices as part of `buildServerApp`, rather than each callsite having to determine the device and pass in this list.